Sources/Sentry/SentryCrashWrapper.m

```diff
@@ -53,11 +53,13 @@ - (BOOL)isApplicationInForeground
 
 - (void)installAsyncHooks
 {
+    NSAssert(false, @"`installAsyncHooks` should not be called");
```

Comment:
`m`: Please explain why it shouldn't be called. Please also apply to the NSAssert below. Maybe you could also explain in a comment why we kept the code.
```suggestion
    NSAssert(false, @"`installAsyncHooks` should not be called, as its behavior is unpredictable and sometimes resulted in unexpected errors.");
```